contributor/es/04-mechanics-of-contributing-article-es.asciidoc

```diff
@@ -0,0 +1,158 @@
+No hace ningún bien a ninguna de las partes que le des vueltas a un problema que podría resolverse en una conversación de unos minutos con alguien más informado sobre el proyecto.
+Está bien pedir ayuda.
+
+Sin embargo, hay una diferencia clave, que supone una ventaja para ti y para otras personas en el futuro:
+En casi todos los casos deberías preferir los canales de comunicación oficiales del proyecto, que pueden ser una lista de correo, una sala de chat, un gestor de incidencias o algo similar, dependiendo del propósito de tener una forma más sincrónica o asincrónica de interactuar, o de las distintas necesidades de estructuración de la comunicación.
+Todos ellos suelen tener en común que se basan en el texto, se archivan, se pueden buscar y vienen con enlaces estables: esto significa que tu pregunta y la respuesta quedarán escritas, y las referencias que enlaces en esas respuestas también se mantendrán accesibles.
+De este modo, podrá beneficiarse de estos conocimientos documentados de forma pasiva en su búsqueda Y ayudar a los futuros colaboradores a tener la misma ventaja.
+Esta documentación pasiva podría incluso servir para enriquecer la documentación "oficial", en caso de que contenga joyas especialmente valiosas, como definiciones importantes que se crearon ad hoc.
+
+Mientras trabajas, si encuentras documentación faltante (o desactualizada), hazle un favor al siguiente Colaborador y actualízala con lo que has descubierto.
+Los equipos de proyecto suelen estar encantados de recibir adiciones, actualizaciones o correcciones para su documentación existente: ¡acabas de encontrar otra oportunidad para contribuir!
+(O simplemente proporciónales amablemente un comentario sobre tu experiencia, y lo que te habría ayudado).
+
+==== Elaboración del código
+
+Todos tenemos nuestras preferencias y opiniones sobre el estilo del código, la sangría, etc.
+El proyecto del equipo anfitrión también las tiene.
+Trata de adaptarte y de coincidir con esas preferencias, incluso si no es lo que harías normalmente, e incluso si no está especificado en el proyecto https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[_`CONTRIBUTING.md`_].
+Si no está seguro, siempre puede preguntar amablemente. No obstante, una contribución de invitado para una característica o una corrección de errores no es el momento de introducir una nueva forma de estructurar o formatear el código del proyecto.
+
+=== Enviar el pull request
+
+Has completado todo el trabajo esencial, has descubierto todas las peculiaridades del problema y del proyecto al que estás contribuyendo, se acerca el momento en el que has planeado que se utilice la nueva característica y quieres asegurarte de que tu contribución se fusione de la forma más rápida y fluida posible.
+
+Esto es lo que puedes hacer para que la revisión y la fusión sean lo más fácil posible para el https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] y el equipo anfitrión.
```

Comment:
I saw down below the suggestion of translating ´merge´ as 'integración' instead of 'fusión' and i think it fits better
```suggestion
Has completado todo el trabajo esencial, has descubierto todas las peculiaridades del problema y del proyecto al que estás contribuyendo, se acerca el momento en el que has planeado que se utilice la nueva característica y quieres asegurarte de que tu contribución se integre de la forma más rápida y fluida posible.

Esto es lo que puedes hacer para que la revisión y la integración sean lo más fácil posible para el https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] y el equipo anfitrión.
```